src/main/java/org/sonar/generic/metrics/GenericMetricMeasureComputer.java

```diff
@@ -26,40 +28,76 @@ public MeasureComputerDefinition define(MeasureComputerDefinitionContext context
 
   @Override
   public void compute(MeasureComputerContext context) {
+    if (context.getMeasure(metric.key()) != null)

```

Comment:
Generic Metric Sensor adds project measures to the context, this check is to make sure that the computer will not calculate project measures (making the sensor do it)

Reply:
This doesn't make sense. If your intention is to ignore project metrics, you should add a break in case  PROJECT. This will break the case where you want to compute a sum/average instead of using the provided metrics in the json file. 

I think you are missing a test for a measure computer.
- add a metric which expects the project-level metric to be calculated
- add a metric which expects the project-level metric to be provided in the json file.

Assert if the project metric has the expected value. This will probably highlight a bug here.